apps/docs/gatsby-config.js

```diff
@@ -12,7 +12,7 @@ module.exports = {
       resolve: '@primer/gatsby-theme-doctocat',
       options: {
         repoRootPath: '../',
-        defaultBranch: 'main',
+        defaultBranch: 'main/apps',
```

Comment:
What does this do @josepmartins?

Reply:
This fixes the edit URL links on the bottom of each page in the docs, as they now points to `/primer/brand/edit/main...` instead of `/primer/brand/apps/edit/main...`. 

<img width="710" alt="Screenshot 2022-12-15 at 09 50 02" src="https://user-images.githubusercontent.com/912236/207814775-e820679f-9451-4664-9170-8aeaf7475c6f.png">

This comes from `Doctocat` taking the `defaultBranch` data to generate the link: https://github.com/primer/doctocat/blob/main/theme/gatsby-node.js#L114

Reply:
ohh interesting. I wouldn't have guessed that. Thanks ❤️ .. all links around docs seems to have been fixed now 🙌